src/stackwalk.c

```diff
@@ -84,8 +84,28 @@ size_t rec_backtrace(uintptr_t *data, size_t maxsize)
     return rec_backtrace_ctx(data, maxsize, &context);
 }
 
+int jl_num_frames(uintptr_t ip, int skipC)
```

Comment:
Prior to adding this it was difficult to estimate how much of the backtrace was from C or from Julia. I'm open to ideas though.